<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestDrawCanvasMainDialog</class>
 <widget class="QDialog" name="QtTestDrawCanvasMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestDrawCanvas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #808080, stop: 1 #ffffff);
}
QPushButton { background-color: rgb(255, 255, 255);  }
QPushButton { color: rgb(0,0,0); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layout_dot">
     <item>
      <widget class="QPushButton" name="button_dot">
       <property name="text">
        <string>Draw dot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_dot_x">
       <property name="text">
        <string>(x =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_dot_x">
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_doy_y">
       <property name="text">
        <string>, y=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_dot_y">
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_dot_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_line">
     <item>
      <widget class="QPushButton" name="button_line">
       <property name="text">
        <string>Draw line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_line_x1">
       <property name="text">
        <string>(x =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_line_x1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_line_y1">
       <property name="text">
        <string>, y=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_line_y1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_line_x2">
       <property name="text">
        <string>)-(x =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_line_x2">
       <property name="value">
        <number>70</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_line_y2">
       <property name="text">
        <string>, y=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_line_y2">
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_line_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_circle">
     <item>
      <widget class="QPushButton" name="button_circle">
       <property name="text">
        <string>Draw circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_circle_x">
       <property name="text">
        <string>(x =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_circle_x">
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_circle_y">
       <property name="text">
        <string>, y=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_circle_y">
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_circle_r">
       <property name="text">
        <string>, radius=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_circle_r">
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_circle_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_ellipse">
     <item>
      <widget class="QPushButton" name="button_ellipse">
       <property name="text">
        <string>Draw ellipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ellipse_x1">
       <property name="text">
        <string>(x1 =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_ellipse_x1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_ellipse_y1">
       <property name="text">
        <string>, y1=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_ellipse_y1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_ellipse_x2">
       <property name="text">
        <string>)-(x2 =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_ellipse_x2">
       <property name="value">
        <number>70</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ellipse_y2">
       <property name="text">
        <string>, y2=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_ellipse_y2">
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ellipse_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_ellipse">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_arc">
     <item>
      <widget class="QPushButton" name="button_arc">
       <property name="text">
        <string>Draw arc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_arc_x1">
       <property name="text">
        <string>(x1 =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_arc_x1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_arc_y1">
       <property name="text">
        <string>, y1=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_arc_y1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_arc_x2">
       <property name="text">
        <string>)-(x2 =</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_arc_x2">
       <property name="value">
        <number>70</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_arc_y2">
       <property name="text">
        <string>, y2=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_arc_y2">
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_arc_start_arc">
       <property name="text">
        <string>, start_arc=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="box_arc_start_arc">
       <property name="minimum">
        <double>-6.280000000000000</double>
       </property>
       <property name="maximum">
        <double>6.280000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_arc_arc_length">
       <property name="text">
        <string>, arc_length=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="box_arc_arc_length">
       <property name="minimum">
        <double>-6.280000000000000</double>
       </property>
       <property name="maximum">
        <double>6.280000000000000</double>
       </property>
       <property name="value">
        <double>3.140000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_arc_close">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_arc">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_color_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="text">
        <string>Color system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="box_color_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>invert</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_coordinat_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="text">
        <string>Coordinat system</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="box_coordinat_system">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>screen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>graph</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_clear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
